src/views/ActivityTab.vue

```diff
@@ -4,7 +4,7 @@
 -->
 
 <template>
-	<div :class="{ 'icon-loading': loading }">
+	<div :class="{ 'activity': true, 'icon-loading': loading }">
```

Comment:
```suggestion
	<div :class="{ 'icon-loading': loading }"
		class="activity">
```

Reply:
Thanks, I forgot it is possible for both can be declared at once.